### DISCUSSION TITLE
Greyhole php

### DISCUSSION THREAD

**Discussion by richardlrzjw** [gboudreau/Greyhole#283] [General]
I am pretty new to Greyhole and wondering if I am using manual installation and configuration, I don't have to have php and web server, am I correct, in this case, I only need to have mysql and samaba installed, am I correct?
My platform is 
Redhat 8
mysql 8
samaba 4

**Comment by gboudreau** [comment]
You need PHP. The INSTALL file will make you install all dependencies needed.
Why do you want to install manually? Why not use the RPM to install?

**Reply by richardlrzjw** [reply]
Thank you, I have tried to follow the installing Greyhole steps, but I got no response after I entered "curl -Ls http://bit.ly/greyhole-package | sudo bash" in the command window.  Therefor, I have to think about download the latest tar.gz file and follow the instruction from the install file.

**Reply by gboudreau** [reply]
You can install the package (rpm) manually like this:

```bash
sudo curl -so /etc/yum.repos.d/greyhole.repo https://www.greyhole.net/releases/rpm/greyhole.repo
sudo yum install php-mysql php-mysqlnd greyhole
```

**Reply by richardlrzjw** [reply]
I have tried couple of times and could implement installation properly by using the command you re commanded.
I got :
[root@Latitude ~]# yum install php-mysql php-mysqlnd greyhole
Updating Subscription Management repositories.
Last metadata expiration check: 0:00:16 ago on Wed 15 Sep 2021 05:35:33 PM EDT.
No match for argument: php-mysql
Error: Unable to find a match: php-mysql

I wonder if it is possible to install Greyhole in Centos or Redhat especially for version 8? what is the best Linux you recommend? ubuntu?

**Reply by gboudreau** [reply]
Just remove `php-mysql` from the list, since it doesn't exist. You need either `php-mysql` or `php-mysqlnd`, and you OS seems to use `php-mysqlnd`:

```
yum install php-mysqlnd greyhole
```
